### PR TITLE
Bug fixes plus 

### PR DESCRIPTION
**Pull request by chrisowensboston**
added "manage_dbserver" Boolean param to profiles/master.pp -- passed along to puppetdb and tells puppetDB whether or not to own the postgresql server. (my use case: I already have a Postgresql server on the host I want to configure; I already manage it; I want puppetdb to create the database but not to take ownership of the server. 

A couple of fixes to get the spec tests to run (listed in CHANGELOG)

Not sure if I was supposed to update CHANGELOG or version number in metadata.json, so please let me know if I should not.

### CODE REVIEW

**Comment by chrisowensboston** [comment]
Something is funky in the travis CI tests - fqdn_rand is supposed to give the same result with the same value of fact $::fqdn, which I think is being set properly in the spec tests for profile::agent, but I'm not getting the same results here on githubb as I'm getting on my own build machine....

**Comment by chrisowensboston** [comment]
PB this looks ready to go from my end if you want to review it for sanity.

File: spec/classes/puppet_profile_master_spec.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@
         {
           "operatingsystem" => "CentOS",
           "operatingsystemrelease" => [
+            "6",
```
Comment (rendhalver) [inline spec/classes/puppet_profile_master_spec.rb]: I definitely haven't tested this class on CentOS 6.
Can you confirm that it actually work on CentOS 6?

**Comment by rendhalver** [comment]
Can you rebase my latest changes and add your changelog enties intothe 2.3.0-alpha section?
Thanks.

**Comment by chrisowensboston** [comment]
It’ll be a couple of days before I can get back to this but will do.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
+##2016-09-28 - Chris Owens <cowens@calmenergyinc.com> 2.2.2
```
Comment (chrisowensboston) [inline CHANGELOG.md]: OK should be fixed

File: spec/classes/puppet_profile_master_spec.rb (hunk 1 above)
Comment (chrisowensboston) [inline spec/classes/puppet_profile_master_spec.rb]: I have tested it on Centos 6

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+Add support for splay and splaylimit parameters in puppet.conf @paulseward
+Fix syntax error on readme @jordigg
+in manifests/repo.pp added include ::puppet, to fix unknown variable puppet::manage_repos @chrisowensboston
+in manifests/init.pp added "+ 0" after calls to fqdn_rand, to force conversion to integer. @chrisowensboston
+in spec/classes/puppet_agent_spec.rb, Changed tests for interpolated minute values to accommodate different versions of fqdn_rand using different entropy sources. Changed host name from "testy" to "testy2" because "testy" happened to give a 0 using fqdn_rand(14)
+In .fixtures.yml, changed puppetlabs/concat version requirement from '1.2.0' to '>= 1.2.2 < 2.0.0', to fix "uknown variable ::is_pe" bug @chrisowensboston
+in Gemfile, forced json_pure to be < 2.0.1 when ruby < 2.0 @chrisowensboston
+in .travis.yml, added a line for rvm 2.2.0, puppet gem 4.7.0, facter gem 2.4.6, hiera gem 3.2.1 @chrisowensboston
+
+
```
Comment (rendhalver) [inline CHANGELOG.md]: Awesome.
Thanks!

File: Gemfile [hunk 4]
```diff
@@ -54,6 +54,10 @@ end
 #   gem 'beaker-puppet_install_helper', :require => false
 # end
 
+# puppet requires json_pure; json_pure > 2.0.1 requires Ruby 2
+# so force the older json_pure in the case of older ruby
+gem 'json_pure', '< 2.0.2', :require => false, :platforms => [:ruby_18, :ruby_19]
+
```
Comment (rendhalver) [inline Gemfile]: I was wondering if this was possible.
Thanks you!

**Comment by chrisowensboston** [comment]
Pete I don’t know GitHub innards well… can I delete my fork of this repo now, without messing up the history of the pull request, etc. on your end?

**Comment by rendhalver** [comment]
You can safely delete your fork.
Your fork is a complete copy of this repo and when I merged it I pulled in the branch and changes you added.
If you want to add further changes later on you will need to make a new fork.
I tend to keep my forks around for this purpose.
You need to sync them up before you do that though.
The lovely github people provide some docs for doing just that here: https://help.github.com/articles/syncing-a-fork/

**Pull request merged**